Gemfile

```diff
@@ -8,14 +8,14 @@ gem 'bump', require: false
 gem 'pry'
 gem 'rake', '~> 12.0'
 gem 'rspec', '~> 3.7'
-gem 'rubocop-rspec', '~> 1.20.0'
+gem 'rubocop-rspec', '~> 1.22.0'
 gem 'simplecov', '~> 0.10'
 gem 'test-queue'
 gem 'yard', '~> 0.9'
 
 group :test do
   gem 'codeclimate-test-reporter', '~> 1.0', require: false
-  gem 'public_suffix', '~> 2.0', require: false
+  gem 'public_suffix', '~> 3.0', require: false
```

Comment:
I guess we can just remove `public_suffix` from the Gemfile.
The gem has been specified from #4679 because the latest version does not support Ruby 2.0.0, and RuboCop supported Ruby 2.0.0. But RuboCop does not support Ruby 2.0.0 already, so we can remove the gem from the file.

Reply:
Sounds good! Updated.